libraries/AP_Notify/Buzzer.cpp

```diff
@@ -66,6 +67,18 @@ void Buzzer::update_pattern_to_play()
         return;
     }
 
+    // initializing?
+    if (_flags.gyro_cal != AP_Notify::flags.gyro_cal) {
+        _flags.gyro_cal = AP_Notify::flags.gyro_cal;
+        play_pattern(INIT_GYRO);
```

Comment:
not sure if we should play buzzer during gyro cal as it could mess it ?

Reply:
moved it to right after the cal completes...still serves the same purpose to let the user know when he can/cannot move plane during ground start